presto-product-tests/conf/presto/etc/catalog/hive.properties

```diff
@@ -7,7 +7,7 @@
 
 connector.name=hive-cdh5
 hive.metastore.uri=thrift://hadoop-master:9083
-hive.metastore.thrift.client.socks-proxy=hadoop-master:1080
+hive.metastore.thrift.client.socks-proxy=hadoop-master:1180
```

Comment:
Sounds good. I'll wait for you to publish the new image. If you want to take over the PR, feel free.